airbyte-integrations/connectors/source-python-http-tutorial/source_python_http_tutorial/spec.json

```
@@ -8,12 +8,14 @@
     "additionalProperties": false,
     "properties": {
       "start_date": {
+	"title": "Start Date"
```

```suggestion
	        "title": "Start Date",
```
Please keep the structure (so that everything is in one column)